server/src/test/java/org/opensearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -1804,82 +1803,43 @@ public void onFailure(final Exception e) {
                 final SetOnce<RepositoriesService> repositoriesServiceReference = new SetOnce<>();
                 repositoriesServiceReference.set(repositoriesService);
                 FileCacheCleaner fileCacheCleaner = new FileCacheCleaner(nodeEnv, null);
-                if (FeatureFlags.isEnabled(FeatureFlags.EXTENSIONS)) {
```

Comment:
I was pretty baffled when I saw this feature flag being checked inside of test code - I'm assuming this was a mistake but would love more detail.

Reply:
I just added them everywhere that extensions code interacted with code outside the extensions directory.

Reply:
I also didn't want to change the initial method to add an extensionsManager parameter, so I overloaded the constructor.